homeassistant/components/waterkotte_heatpump/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "invalid_auth": "Invalid authentication",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Host",
+                    "password": "Password",
+                    "username": "Username"
+                }
+            }
+        }
+    }
+}
```

Comment:
I believe this should no longer be pushed

Reply:
That is correct

Reply:
redolved with [eab7ef6](https://github.com/home-assistant/core/pull/88140/commits/eab7ef65b5f3a23e0837922b0fbdb496ad21f652)